root/static/scripts/common/MB/Control/Autocomplete.js

```diff
@@ -723,7 +723,9 @@ MB.Control.autocomplete_formatters = {
         _(item.labels)
             .groupBy(getLabelName)
             .each(function (releaseLabels, name) {
-                var catalogNumbers = _(releaseLabels).map(getCatalogNumber).compact().sort().value();
+                var catalogNumbers = _(releaseLabels).map(getCatalogNumber).compact()
+                    .sort()
+                    .value();
```

Comment:
Hi!

This comment is not specifically related to the rule (which thinks this is just fine), but while we have it set up to allow chains of up to 3 elements, we'd prefer to have *everything* in its own line if it's long enough to be split. So, a small chain stays in one line, but this one becomes: 

```
                var catalogNumbers = _(releaseLabels)
                    .map(getCatalogNumber)
                    .compact()
                    .sort()
                    .value();
```
Could you make those changes? Sorry, we thought that's how the rule worked by default, but it turns out it does not :smile: 

Same applies to a lot of other cases in this PR :)